### DISCUSSION TITLE
Variable Fonts support

### DISCUSSION THREAD

**Discussion by tastez** [cables-gl/cables_docs#287] [Feature Requests] [answered]
would be cool to load up fonts and have all of the variables exposed for adjustment.

some links
https://www.axis-praxis.org/samsa/

https://github.com/Lorp/samsa

https://fontgauntlet.com/

**Comment by pandrr** [comment, marked as answer]
its supported via css properties, otherwise not possible

**Comment by pandrr** [comment]
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide

https://cables.gl/p/oYpE4o
this canvas technique works in chrome, not firefox

**Comment by tastez** [comment]
right via CSS is perfect
